lib/srv/usermgmt_test.go

```diff
@@ -253,6 +254,7 @@ func TestUserMgmtSudoers_CreateTemporaryUser(t *testing.T) {
 	users := HostUserManagement{
 		backend: backend,
 		storage: pres,
+		log:     slog.Default(),
```

Comment:
We can use `github.com/gravitational/teleport/lib/utils.NewSlogLoggerForTests()` instead.